src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicClientTransformer.java

```diff
@@ -247,10 +244,10 @@ private StaticLangApiView generateApiClass(GapicInterfaceContext context) {
     apiClass.name(namer.getApiWrapperClassName(context.getInterfaceConfig()));
     apiClass.implName(namer.getApiWrapperClassImplName(context.getInterfaceConfig()));
     apiClass.grpcServiceName(namer.getGrpcContainerTypeName(context.getInterfaceModel()));
-    String grpcTypeName = namer.getGrpcServiceClassName(context.getInterfaceModel());
-    int dotIndex = grpcTypeName.indexOf('.');
-    apiClass.grpcTypeNameOuter(grpcTypeName.substring(0, dotIndex));
-    apiClass.grpcTypeNameInner(grpcTypeName.substring(dotIndex + 1, grpcTypeName.length()));
+    String rpcTypeName = namer.getGrpcServiceClassName(context.getInterfaceModel());
+    int dotIndex = rpcTypeName.indexOf('.');
+    apiClass.grpcTypeNameOuter(rpcTypeName.substring(0, dotIndex));
+    apiClass.grpcTypeNameInner(rpcTypeName.substring(dotIndex + 1, rpcTypeName.length()));
```

Comment:
Maybe revert the renaming of this variable

Reply:
Done.